specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -4459,6 +4975,149 @@
           "type": "string"
         }
       }
+    },
+    "WorkloadGroupProperties": {
+      "description": "Workload group definition. For more information look at sys.workload_management_workload_groups (DMV).",
+      "required": [
+        "minResourcePercent",
+        "maxResourcePercent",
+        "minResourcePercentPerRequest"
+      ],
+      "type": "object",
+      "properties": {
+        "minResourcePercent": {
+          "format": "int32",
+          "description": "The workload group minimum percentage resource.",
+          "type": "integer"
+        },
+        "maxResourcePercent": {
+          "format": "int32",
+          "description": "The workload group cap percentage resource.",
+          "type": "integer"
+        },
+        "minResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request minimum grant percentage.",
+          "type": "number"
+        },
+        "maxResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request maximum grant percentage.",
+          "type": "number"
+        },
+        "importance": {
+          "description": "The workload group importance level.",
+          "type": "string"
+        },
+        "queryExecutionTimeout": {
```

Comment:
Please specify what units this uses in the property name and/or description

Reply:
Will change the description to "description": "The workload group query execution timeout in seconds.",